specification/sections/01-flag-evaluation.md

```diff
@@ -33,6 +33,14 @@ See [provider](./02-providers.md) for details.
 
 #### Requirement 1.1.3
 
+> The `API` **MUST** provide a function to bind a given `provider` to one or more client `name`s.
+
+```java
+OpenFeature.setProvider("client-name", new MyProvider());
+```
+
+#### Requirement 1.1.4
+
```

Comment:
I'm interested in why you chose to go with the "number shifting" rather than add this to the end of this section (I think this is fine, I'm just wondering if you came to any conclusions).

Also, have you given any more thought to non-numeric requirements?

Reply:
I would like to avoid shifting spec version numbers if possible. @justinabrahms, would it be possible to add the changes to new sections? If not, we'll want to make sure SDK maintainers update test names based on the new requirement numbers.

Reply:
non-numeric requirements are out of scope for this change. I added this where I did so that it's numerically in order and located along side the other "set the provider" spec stuff. I could also name this `1.1.3-b` or similar, if you prefer?

Reply:
No, that's ok. Your reasoning makes sense to me. Thanks for clarifying!